plain-pipelines.d/paas-product-page.yml

```diff
@@ -0,0 +1,100 @@
+---
```

Comment:
I don't think we want to differentiate "plain" pipelines and create a separate directory for them. I suggest to put this into the pipelines directory

Reply:
The reason is that the `plain-pipelines.d` is processed automatically by the setup.yml itself. But maybe I can rethink it, and we upload them in the `make pipelines` itself :thinking:

Reply:
I would go for the latter.